resources/definitions/ultimaker_original.def.json

```diff
@@ -18,7 +18,9 @@
         "machine_extruder_trains":
         {
             "0": "ultimaker_original_extruder_0"
-        }
+        },
+        "firmware_file": "MarlinUltimaker-{baudrate}.hex",
+        "firmware_hbk_file": "MarlinUltimaker-HKB-{baudrate}.hex"
```

Comment:
Looks like it should be `hkb` because of `HKB`.

Reply:
"hbk" is "heated bed kit", and the filename refers to files in the cura-binary-data repo. Thanks for spotting the typo.